hack/make-release.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+# set vars 
+if [ -n "$1" ]; then
```

Comment:
- use `[[` not `[`

- make sure you document usage of script simply at the beginning of the script.